ts/features/payments/common/utils/index.ts

```diff
@@ -123,23 +123,30 @@ export const getSortedPspList = (
   pspList: ReadonlyArray<Bundle>,
   sortType: WalletPaymentPspSortType
 ) => {
-  switch (sortType) {
-    case "name":
-      return _.orderBy(pspList, psp => psp.pspBusinessName);
-    case "amount":
-      return _.orderBy(
-        pspList,
-        ["taxPayerFee", "pspBusinessName"],
-        ["asc", "asc"]
-      );
-    case "default":
-    default:
-      return _.orderBy(
-        pspList,
-        ["onUs", "taxPayerFee", "pspBusinessName"],
-        ["desc", "asc", "asc"]
-      );
-  }
+  const onUsBundle = pspList.find(psp => psp.onUs);
+  const otherBundles = pspList.filter(psp => !psp.onUs);
+
+  const sortedBundles = (() => {
+    switch (sortType) {
+      case "name":
+        return _.orderBy(
+          otherBundles,
+          ["pspBusinessName", "taxPayerFee"],
+          ["asc", "asc"]
+        );
+      case "amount":
+        return _.orderBy(
+          otherBundles,
+          ["taxPayerFee", "pspBusinessName"],
+          ["asc", "asc"]
+        );
+      case "default":
+      default:
+        return otherBundles;
+    }
+  })();
+
+  return onUsBundle ? [onUsBundle, ...sortedBundles] : sortedBundles;
```

Comment:
Considering the comment left by [the backend team](https://pagopa.atlassian.net/browse/IOBP-833?focusedCommentId=183749), we shouldn't do any override of the sorting when the sortType is `default` because is managed by the backend how to handle the default case, so we need to remove this check without setting every time the onUs as the first element. Let me know if you have any questions about this.